x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/request/RequestUtils.java

```diff
@@ -29,6 +29,14 @@ public static String bearerToken(String apiKey) {
         return "Bearer " + apiKey;
     }
 
+    public static Header createApiKeyHeader(SecureString apiKey) {
```

Comment:
Nit: consistency with `createAuthBearerHeader()`.
```suggestion
    public static Header createAuthApiKeyHeader(SecureString apiKey) {
```